interviewing/technical-interviewing.md

```diff
@@ -25,6 +25,11 @@ on solving technical questions posed by an interviewer instead of
 trying to recall a basic concept of programming, the more confident you
 will feel and the more successful you'll be.
 
+**Which companies focuses on technical and coding skills?**
+- [Amazon](https://www.makeuseof.com/tag/si-coding-challenges-lead-money-jobs/)
```

Comment:
Can you provide a sentence on what they will find at each resource?

Reply:
@alodahl each of the resources says how technology is used by the companies. So, can I directly mention before the companies: "the below links describe the use of technology in the company"

Reply:
I think this is a weird way to list it, since there is no description about these specific companies in the link.  But there is a description of the hackerrank challenge.  Can you change this to describing how the challenge in the link could lead to jobs at Amazon, Redhat, and Uber?